jenkins/files/sars_cov_2/Jenkinsfile

```diff
@@ -56,8 +56,8 @@ pipeline {
         string(name: 'ANALYSIS_RUN', defaultValue: '', description: 'This can be the test name')
         choice(name: 'SEQUENCING_TECHNOLOGY', choices: ['illumina', 'ont', 'unknown'], description: 'The sequencing technology. Use "unknown" to process fasta files')
         string(name: 'KIT', defaultValue: '', description: 'The version of the artic primer scheme used by the ncov-artic pipeline')
-        string(name: 'CLUSTER_NAME', defaultValue: 'saas-dev.k8s.congenica.net', description: 'The Kubernetes cluster where the Congenica platform is deployed that the tests will run against.')
-        string(name: 'NAMESPACE', defaultValue: 'psga', description: 'The Kubernetes namespace where the Congenica platform is deployed that the tests will run against.')
+        string(name: 'CLUSTER_NAME', defaultValue: 'arn:aws:eks:eu-west-2:566277102435:cluster/psga-dev-eks', description: 'The Kubernetes cluster where the Congenica platform is deployed that the tests will run against.')
```

Comment:
Andzej confirmed this is okay, although it might be renamed for simplicity in the future.